mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/WEB-INF/src/content/Language.properties

```diff
@@ -20,6 +20,7 @@ profile-page-is-not-updated=Profile page is not updated
 profile-problem=Profile problem
 sample-task=Sample Task
 success=Form successfully submitted!
+task-added=Success! Task added to the list
```

Comment:
1. Language keys should be equal to language values, until the first dot.
1. There should be no exclamation marks.
1. Language values should be either full sentences, ending with dot, or capitalized titles.

For example, the above two lines should be something like:
```
form-successfully-submitted=Form successfully submitted.
task-added-to-the-list=Task added to the list.
```